packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -398,6 +398,18 @@ class ToolbarOptions {
 /// methods such as [RenderEditable.selectPosition],
 /// [RenderEditable.selectWord], etc. programmatically.
 ///
+/// {@template flutter.widgets.editableText.showCaretOnScreen}
+/// ## Keep the caret visisble when focused
+///
+/// When focused, this widget will make attempts to keep the text area and its
+/// caret (even when [showCursor] is `false`) visible, on these occasions:
+///
+///  * When the user focuses this text field and it is not [readOnly].
+///  * When the user changes the selection of the text field, or changes the
+///    text when the text field is not [readOnly].
+///  * When the virtual keyboard pops up.
```

Comment:
@goderbauer I saw you added the viewInset-triggered showCaretOnScreen code. Should we do something similar when the phone rotates?

Reply:
What do Android and iOS do natively when you rotate the phone? Do they keep the focused textfield in view? If so, we should probably do something similar.

Reply:
iOS text fields don't seem to do that. There are no open issues either I guess I'll leave it as is.